audit.json

```diff
@@ -1,1156 +1,2317 @@
-  "6_Path Traversal_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=c%3A%2F_GET": "ignore",
-  "10055_CSP: Wildcard Directive_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10055_CSP: Wildcard Directive_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10055_CSP: script-src unsafe-inline_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10055_CSP: script-src unsafe-inline_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10055_CSP: style-src unsafe-inline_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10055_CSP: style-src unsafe-inline_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "90017_XSLT Injection_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=%3Cxsl%3Avalue-of+select%3D%22system-property%28%27xsl%3Avendor%27%29%22%2F%3E_GET": "ignore",
-  "10055_CSP: Notices_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10055_CSP: Notices_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10010_Cookie No HttpOnly Flag_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10010_Cookie No HttpOnly Flag_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10011_Cookie Without Secure Flag_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10011_Cookie Without Secure Flag_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10054_Cookie without SameSite Attribute_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10054_Cookie without SameSite Attribute_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10027_Information Disclosure - Suspicious Comments_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10109_Modern Web Application_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10104_User Agent Fuzzer_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore"
+<<<<<<< HEAD
```

Comment:
this file seems like it was corrupted in a merge conflict, does it need to change?